### DISCUSSION TITLE
Как выбрать другое оружие?

### DISCUSSION THREAD

**Discussion by zoosvit** [devapromix/lizardry#120] [Q&A] [answered]
Это наверное ошибка в игре. При регистрации выбрал молот войны (у него самый крутой дамаг). А в игре в руках меч. У торговца в игре одни только мечи. И лутятся из оружки тоже только мечи.

**Comment by devapromix** [comment, marked as answer]
Привет! Я еще не полностью доделал регистрацию. Пока есть одноручные мечи и бронь из кожи. Много чего из запланированного еще не реализовано.
